examples/README.md

```diff
@@ -0,0 +1,133 @@
+# DIB (Docker Image Builder) Example Configuration Guide
+
+This guide explains how to configure and use the DIB example for building and pushing Docker images using Kubernetes.
+
+## Overview
+
+The example in this directory demonstrates how to use DIB to build Docker images using Buildkit in a Kubernetes cluster. The example includes:
+
+- A Dockerfile for creating a custom Buildkit image
+- A buildkitd-launcher script for starting the Buildkit daemon
+- A .dib.yaml configuration file for configuring the build process
+
+## Building and Pushing the Image
+
+The first step is to build and push the custom Buildkit image defined in the Dockerfile:
+
+```bash
+# Navigate to the examples directory
+cd examples
+
+# Build the image
+docker build -t <your-registry>/custome-buildkit:latest .
+
+# Push the image to your registry
+docker push <your-registry>/custome-buildkit:latest
+```
+
+Replace `<your-registry>` with your Docker registry URL. For example, using the registry from the .dib.yaml file:
+
+```bash
+docker build -t europe-west9-docker.pkg.dev/radio-france-dev/registry-test/custome-buildkit:latest .
+docker push europe-west9-docker.pkg.dev/radio-france-dev/registry-test/custome-buildkit:latest
```

Comment:
This Docker registry is private and cannot be referenced in documentation

```suggestion
docker build -t <your-registry>/custome-buildkit:latest .
docker push <your-registry>/custome-buildkit:latest
```

Reply:
@Thibaut-gauvin  fixed